tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py

```diff
@@ -83,7 +102,22 @@
     # [END how_to_cloud_dataproc_delete_batch_operator]
     delete_batch.trigger_rule = TriggerRule.ALL_DONE
 
-    create_batch >> get_batch >> list_batches >> delete_batch
+    # [START how_to_cloud_dataproc_delete_batch_operator]
```

Comment:
duplicated doc marker - we don't need the second example of deleting batch in the docs, it looks trivial enough

Reply:
Done